launcher/client/src/App.tsx

```diff
@@ -32,7 +32,7 @@ import { useSigner, useChainId } from 'wagmi';
 import { ChainId, ESCROW_NETWORKS } from './constants';
 import { useAuth } from './hooks/auth';
 import { AppContext } from 'src/state';
-import { goToTab } from 'src/state/actions';
+import { goToTab, changeLauncerStatus } from 'src/state/actions';
```

Comment:
This is throwing an error. Plz check. You will need to fix the typo here as well.